package.json

```diff
@@ -45,10 +46,12 @@
 		"p-map": "^4.0.0"
 	},
 	"devDependencies": {
+		"@types/node": "^14.14.25",
 		"ava": "^2.4.0",
 		"execa": "^4.0.0",
 		"file-type": "^14.3.0",
 		"tempy": "^0.5.0",
+		"tsd": "^0.14.0",
 		"xo": "^0.30.0"
```

Comment:
Upgrade to the latest version and it will help you with some of the style issues.

Reply:
Got it